coral/src/domain/topic/topic-api.ts

```diff
@@ -171,6 +171,14 @@ const declineTopicRequest = (payload: DeclineTopicRequestPayload) => {
   >(`/request/decline`, payload);
 };
 
+type DeleteTopicRequestPayload = RequestVerdictApproval<"TOPIC">;
+const deleteTopicRequest = (payload: DeleteTopicRequestPayload) => {
```

Comment:
I know we are doing it in the wayb currently implemented elsewhere, but I'm wondering if we can change these functions to only take `reqIds`. Then we hardcode `requestEntityType: "TOPIC"` in the `payload`. Because having a parameter that can only be a single value feels a little bit superfluous :o

Are there considerations that would make this an unwise choice?

We would need to refactor the approve/decline request handlers as well if we decide that the approach I describe here (that @programmiri already suggested, I believe) is correct.

Reply:
I've done it like that for schemas: 

```
const approveSchemaRequest = ({
  reqIds,
}: {
  reqIds: RequestVerdictApproval<"SCHEMA">["reqIds"];
}) => {
  return api.post<
    KlawApiResponse<"approveRequest">,
    RequestVerdictApproval<"SCHEMA">
  >(`/request/approve`, {
    reqIds,
    requestEntityType: "SCHEMA",
  });
};


const declineSchemaRequest = ({
  reqIds,
  reason,
}: Omit<RequestVerdictDecline<"SCHEMA">, "requestEntityType">) => {
  return api.post<
    KlawApiResponse<"declineRequest">,
    RequestVerdictDecline<"SCHEMA">
  >(`/request/decline`, {
    reqIds,
    reason,
    requestEntityType: "SCHEMA",
  });
};
```

Reply:
Should we refactor them in one go?